ONSdigital/eq-survey-runner
# google analytics issue fix

### What is the context of this PR?
Fixs #977 Google analytics events were behaving erratically so this fixes the issue.

### How to review 
- in terminal `set -x EQ_UA_ID UA-56892037-7`
- Select schema `census_household.json`
- navigate to question that doesn't have a label and click on `guidance`, `help and support`, `previous`.
- cause an error on the page
- check google analytics
